src/storage/FirebaseStorageDirective.js

```diff
@@ -0,0 +1,28 @@
+(function () {
+  "use strict";
+
+  function FirebaseStorageDirective($firebaseStorage, firebase) {
+    return {
+      restrict: 'A',
+      priority: 99, // run after the attributes are interpolated
+      scope: {},
+      link: function (scope, element, attrs) {
+        // $observe is like $watch but it waits for interpolation 
+        // Ex: <img firebase-src="{{ myUrl }}"/> 
+        attrs.$observe('firebaseSrc', function (newVal) {
+          if (newVal !== '' && newVal !== null && newVal !== undefined) {
```

Comment:
I'm not sure I understand how this directive should work.  It appears the directive's purpose is to generate the url to the file based on a path to the file.  This path looks to be provided by the `gs-url` attribute.  The code though is setup to watch the value `myUrl` passed in by the user.  What exactly is the user suppose to be passing to the directive?  It appears the directive should be using the observed value of the `firebase-src` attribute instead of the `gs-url` attribute.